dev-support/test-patch.d/builtin-personality.sh

```diff
@@ -14,148 +14,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-function builtin_personality_modules
-{
-  local repostatus=$1
-  local testtype=$2
-
-  local module
-
-  yetus_debug "Using builtin personality_modules"
-  yetus_debug "Personality: ${repostatus} ${testtype}"
-
-  clear_personality_queue
-
-  # this always makes sure the local repo has a fresh
-  # copy of everything per pom rules.
-  if [[ ${repostatus} == branch
-     && ${testtype} == mvninstall ]];then
-     personality_enqueue_module .
-     return
-   fi
-
-  for module in ${CHANGED_MODULES}; do
-    # shellcheck disable=SC2086
-    personality_enqueue_module ${module}
-  done
-}
-
 function personality_modules
 {
-  builtin_personality_modules "$@"
-}
-
-function builtin_mvn_personality_file_tests
-{
-  local filename=$1
-
-  yetus_debug "Using builtin mvn personality_file_tests"
-
-  if [[ ${filename} =~ src/main/webapp ]]; then
-    yetus_debug "tests/webapp: ${filename}"
-  elif [[ ${filename} =~ \.sh
-       || ${filename} =~ \.cmd
-       || ${filename} =~ src/main/scripts
-       || ${filename} =~ src/test/scripts
-       ]]; then
-    yetus_debug "tests/shell: ${filename}"
-  elif [[ ${filename} =~ \.md$
-       || ${filename} =~ \.md\.vm$
-       || ${filename} =~ src/site
-       || ${filename} =~ src/main/docs
```

Comment:
shellcheck: 3: note: Double quote to prevent globbing and word splitting. [SC2086]